spec/helpers/factory_helpers.rb

```diff
@@ -65,5 +65,5 @@ def metadata_element
         <div data-type="subject" id="subject">Subject</div>
       </div>
     HTML
-  )
+  ).metadata
```

Comment:
why this change?

Reply:
Specs fail without this change -- several functions (i.e. `BakeChapterGlossary.v1`)  have been updated to expect a `MetadataElement` instead of an `Element`. This function is called in the specs to generate metadata, so it must now also return a `MetadataElement`.

Reply:
ah right